src/Stream/test/BinderRabbitMQ.Test/RabbitBinderTests.cs

```diff
@@ -1843,12 +1892,26 @@ public void TestPolledConsumerWithDlqRePub()
             binding.Unbind();
         }
 
-        [Fact]
-        public void TestAnonymousGroup()
+        // TestCustomBatchingStrategy - Test not ported because CustomBatching Strategy is not yet supported in Steeltoe
+        protected override string GetEndpointRouting(object endpoint)
+        {
+            var spelExp = GetPropertyValue<SpelExpression>(endpoint, "RoutingKeyExpression");
+            return spelExp.ExpressionString;
+        }
+
+        protected override void CheckRkExpressionForPartitionedModuleSpEL(object endpoint)
         {
-            TestAnonymousGroupBase();
+            var routingExpression = GetEndpointRouting(endpoint);
+            var delimiter = GetDestinationNameDelimiter();
+            var dest = GetExpectedRoutingBaseDestination($"'part{delimiter}0'", "test") + " + '-' + Headers['" + BinderHeaders.PARTITION_HEADER + "']";
+
+            Assert.Contains(dest, routingExpression);
         }
 
+        protected override string GetExpectedRoutingBaseDestination(string name, string group) => name;
+
+        protected override bool UsesExplicitRouting() => true;
+
         private void TestAutoBindDLQPartionedConsumerFirstWithRepublishGuts(bool withRetry)
```

Comment:
This looks like it's used by 2 tests, one passes true and the other false - should the combination be converted to a theory with inline data instead?

Reply:
I thought of it, but it seemed best to keep name parity with the tests on the spring cloud stream binder side.